### ISSUE TITLE
using Reexport?

### ISSUE THREAD

**Issue by EricForgy** [JuliaML/StochasticOptimization.jl#2]
I am getting an error with `using Learn` ( https://github.com/JuliaML/Learn.jl/issues/1 ). I can trace the error back to here:

https://github.com/JuliaML/StochasticOptimization.jl/blob/master/src/StochasticOptimization.jl#L4

I can see a `@reexport using LearnBase`. Does there need to be a `using Reexport` there. If so, I can submit a PR.

If I first do a `using Reexport` and then `using Learn` everything goes through.

**Comment by tbreloff** [comment]
Eric: glad you're excited to give these a try, but they are not ready for
publuc consumption yet. This package in particular is in a pre-pre-alpha
state. No need to fix small errors. This code will be rewritten a few times
I'm sure

**Comment by Evizero** [comment]
Maybe this should be more obvious in Learn.jl.

**Comment by tbreloff** [comment]
Agreed. It's a good place to put package statuses.

**Comment by EricForgy** [comment]
Hi @tbreloff 

Thanks for the status update. Although there is no need to fix small errors at this point, I don't see a reason why I couldn't help even if things get overwritten :)

I don't mind if this isn't ready for public consumption. Although I'm still learning Julia, I've been doing this kind of stuff for more than 20 years (computational physics, differential geometry, algebraic topology) and can handle rough edges and happy to contribute - even develop core pieces if it makes sense :+1:

**Comment by tbreloff** [comment]
Hey if you're ready to get your hands dirty that's great.  We're all spread
a little thin and could use the help. You should probably read the past
issues and discussions, including in the roadmap, and join the gitter chat.
Do you have any particular area that you're interested in contributing to?

**Comment by EricForgy** [comment]
Hi @tbreloff 

I joined the Gitter room for JuliaML/chat :+1: 

Just this Friday, I was on a panel discussing "Artificial Intelligence in Investment Management". That is me in the middle on the couch :) 

![image](https://cloud.githubusercontent.com/assets/8954730/18228570/3a6f101e-7287-11e6-9cba-017ccaee9c95.png)

There were more than 100 people in attendance and the response was super. There is a lot of interest in the topic. So I am interested in application of machine learning to financial services whether it is "robo advisor" or "chat bot" etc.

I've found a number of well-written introductory articles with source code provided in Python and Lua, but nothing yet in Julia. I hope to work through some samples and convert them to Julia as an exercise.
